tox.ini

```diff
@@ -1,9 +1,11 @@
 [tox]
-envlist = linting, doclinting, cov-init, py{36,37,38,39}, cov-report, bench, mypy
+envlist = linting, doclinting, cov-init, py{36,37,38,39}, dbt01{7,8}-py39, cov-report, bench, mypy

```

Comment:
Perhaps selfish here, but locally I do most of my development in py38. Can you change `dbt01{7,8}-py39` to `dbt01{7,8}-py{38,39}` so that I'm using one of our standard test environments?

Reply:
dbt doesn't actually work with py39 https://github.com/fishtown-analytics/dbt/issues/2865 so I switched the tests back to py38